content/ru/contributor-ladder/_index.md

```diff
@@ -0,0 +1,126 @@
+**Все эти роли одинаково важны** и способствуют процветанию сообщества. 
+Пожалуйста, обратитесь к разделам [Как внести свой вклад](/ru/contribute/) и [Руководство по стилю](/ru/style-guide/), чтобы подробнее узнать о том, как поучаствовать в проекте.
+
+## 2. Апруверы
+
+Апруверы дают обратную связь по PR'ам и одобряют их. Любой активный участник может стать апрувером (см. [Стать апрувером](#стать-апрувером)). 
+В Глоссарии различают два вида апруверов: (1) апруверы для оригинальной английской версии глоссария и (2) апруверы для команд локализации.
+
+Апруверы глоссария должны:
+
+- Проверять PR на техническую точность; 
+- Закреплять Issues за авторами и навешивать на них соответствующие лейблы;
+- Давать контрибьюторам обратную связь и направлять их при необходимости,
+- Вычитывать и редактировать материалы.
+
+Если апрувер более не заинтересован в выполнении вышеперечисленных обязанностей или не может их выполнять, он должен сообщить об этом мейнтейнерам и сложить свои полномочия.
+
+### Апруверы английской версии Глоссария
+
+Различают три вида апруверов:
+
+1) Апруверы с богатым техническим бэкграундом.
+2) Апруверы, умеющие хорошо и понятно писать.
+3) Апруверы, владеющие обоими навыками.
+
+========================================================
+
+**Технические апруверы**: Люди с обширными техническими знаниями могут быть апруверами, не обладая хорошими навыками письма на английском языке. 
+Однако, проверяя PR, они не должны ограничиваться его технической стороной — следует убедиться, что его проверит редактор.
+
+**Редакторы**: Редакторы вычитывают термины и следят за тем, чтобы изложение велось простым языком в соответствии с Руководством по стилю. 
+Если в описание термина вносились серьезные изменения, редактор должен попросить технического специалиста просмотреть его еще раз, чтобы убедиться, что смысл сохранился.
+
+### Апруверы локализаций
+
+У глоссария также есть апруверы локализаций, которые принадлежат к одной из команд локализации (команды, занимающейся переводом глоссария). 
+Апруверы локализации занимаются проверкой локализованных терминов и могут мержить PR'ы в соответствующую ветку локализации. 
+Любой апрувер локализации также может апрувером для английской версии глоссария, если он соответствует требованиям. 
+
+### Стать апрувером
+
+Кандидаты в апруверы должны иметь опыт подачи качественных PR'ов и помощи другим в приведении их PR'ов в состояние, пригодное для слияния. 
+Если позволяет часовой пояс, они также должны регулярно посещать встречи [Рабочей группы по глоссарию](https://www.cncf.io/calendar/).
+
+Чтобы стать апрувером, прежде всего расскажите о своем желании мейнтейнерам.
+Они попросят вас продемонстрировать вышеуказанные навыки, внося PR'ы, делая обзоры и выполняя другие подобные задачи под их руководством. 
+Через некоторое время мейнтейнеры решат, давать ли вам статус апрувера. 
+Решение будет основано на показанном вами уровне мастерства и ответственности.
```

Comment:
Personally, I like the idea of _responsibility_ more, but let's stick with the original wording (i.e. _responsiveness_).
```suggestion
Решение будет основано на показанном вами уровне мастерства и отзывчивости.
```